es2022.md

```diff
@@ -83,8 +83,40 @@ extend interface BinaryExpression <: Expression {
 - `left` can be a private identifier (e.g. `#foo`) when `operator` is `"in"`.
 - See [Ergonomic brand checks for Private Fields][proposal-private-fields-in-in] for details.
 
+# Modules
+
+See [Arbitrary module namespace identifier names] for more details.
+
+## Imports
+
+### ImportSpecifier
+
+```js
+extend interface ImportSpecifier <: ModuleSpecifier {
+    imported: Identifier | Literal;
+}
+```
+
+If `imported` is a `Literal`, `imported.value` must be a string without lone surrogate.
+
+## Exports
```

Comment:
Should [ExportAllDeclaration](https://github.com/estree/estree/blob/master/es2020.md#exportalldeclaration) also be extended?

```js
export * as "foo" from "mod"
```

Reply:
Thanks! a22ea89